HealthMonitoring.UnitTests/Domain/EndpointTests.cs

```diff
@@ -37,7 +37,7 @@ public void CheckHealth_should_update_the_endpoint_with_its_health_status()
         [Fact]
         public async void CheckHealth_should_not_update_disposed_endpoint()
         {
-            var endpoint = new Endpoint(Guid.Empty, MonitorMock.Mock("monitor"), "address", "name", "group");
+            var endpoint = new Endpoint(Guid.Empty, MonitorMock.Mock("monitor"), "address", "name", "group", new[] { "t1", "t2" });
 
             var sampler = new Mock<IHealthSampler>();
             var token = new CancellationToken();

```

Comment:
It would be good to see some explicit tests for tags update as I do not see any here.